ui/web/admin_http_assets/app.js

```diff
@@ -92,7 +92,7 @@ app.controller("MainCtl", ["$scope", "$resource", "$modal", function($scope, $re
   $scope.addAggregator = function() {
     $scope.alerts = [];
     $scope.newAgg.$save().then(function(resp) {
-      $scope.newAdd = new Aggregator({Type: "agg", Interval:60, Wait: 120});
+      $scope.newAgg = new Aggregator({Type: "agg", Interval:60, Wait: 120});
```

Comment:
wow, the web ui hasn't seen love in a long time. were you able to actually use it?

Reply:
Yeah, I hadn't either but I did verify that it works.